core/src/main/java/com/linecorp/armeria/server/HttpServerHandler.java

```diff
@@ -371,12 +375,15 @@ private void handleRequest(ChannelHandlerContext ctx, DecodedHttpRequest req) th
         final RoutingResult routingResult = routed.routingResult();
         final ServiceConfig serviceCfg = routed.value();
         final HttpService service = serviceCfg.service();
+        final EventLoop newEventLoop =
+            serviceCfg.serviceWorkerGroup() == null ? eventLoop
+                : serviceCfg.serviceWorkerGroup().next();
 
         final DefaultServiceRequestContext reqCtx = new DefaultServiceRequestContext(
                 serviceCfg, channel, config.meterRegistry(), protocol,
                 nextRequestId(), routingCtx, routingResult,
                 req, sslSession, proxiedAddresses, clientAddress,
-                System.nanoTime(), SystemInfo.currentTimeMicros());
+                System.nanoTime(), SystemInfo.currentTimeMicros(), newEventLoop);
```

Comment:
Should we switch the working event loop if the current event loop differs from `newEventLoop`?